bin/z-gerrit

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ -n "$Z_LIB" ]; then
```

Comment:
This should be in a function that can be shared by all scripts and a function to check `Z_LIB`, `Z_HOME`, etc. to validate the environment is setup properly and then the file just sourced. I think? Thoughts?